alicat/__init__.py

```diff
@@ -19,7 +19,8 @@ def command_line():
                         "to read devices routed through a converter.")
     parser.add_argument('--address', '-a', default='A', type=str, help="The "
                         "device address, A-D. Should only be used if multiple "
```

Comment:
This was there all along, but might as well fix it now.  I believe they go A-Z not A-D

Reply:
Yup it is A-Z! I changed it.